includes/class-sensei-modules.php

```diff
@@ -1054,21 +1058,24 @@ public function calculate_user_module_progress( $user_id = 0, $module_id = 0, $c
 	}
 
 	/**
-	 * Register admin screen for ordering modules
+	 * Register admin pages related to modules.
 	 *
 	 * @since 1.8.0
-	 * @deprecated 4.0.0
 	 *
 	 * @return void
 	 */
 	public function register_modules_admin_menu_items() {
-		_deprecated_function( __METHOD__, '4.0.0' );
```

Comment:
I had a very similar problem with "Order courses" page. So this comment is my attempt to find a better general approach.

I didn't changed and "re-enabled" deprecated method. As we suppose that deprecated method might be invoked outside it might be unexpected that its behavior changed. Because of that I added a new method with a single adding of needed "Order courses" page (`Sensei_Admin::add_course_order` method in https://github.com/Automattic/sensei/pull/4672/files).

How do you think, what is preferable way after all?

Reply:
The `register_modules_admin_menu_items` method was deprecated by #4618 which is not in master yet, so I think it's OK to revert the deprecation.

Reply:
Sorry, now I understand your point, thanks to Donna. I've re-deprecated the method and added a new one for the menu in 7f08a4e8f1cf6265eee38b1862f1e10997601ad8 and acee092473d24b891ed31bca03ce18e6b8e40f6f.

Reply:
@merkushin I've noticed now that you've added the "order modules" submenu in one of your PR's. I guess I should revert acee092473d24b891ed31bca03ce18e6b8e40f6f, right?